obal/data/playbooks/update.yml

```diff
@@ -55,3 +48,13 @@
       with_items: "{{ new_source_urls.stdout_lines | list }}"
       when:
         - "'://' not in item"
+
+    - name: 'Create git branch'
+      command: "git checkout -b {{ inventory_hostname }}-{{ version }}"
```

Comment:
this makes `version` mandatory, but `obal/data/roles/fetch_sources/tasks/main.yml` uses `version` to differentiate between "plain update" (`version is defined`) and "copy from upstream" (`version is not defined`)